hasher-matcher-actioner/hmalib/common/actioner_models.py

```diff
@@ -29,39 +29,35 @@ def __eq__(self, another_label: object) -> bool:
             return NotImplemented
         return self.key == another_label.key and self.value == another_label.value
 
-    def __hash__(self) -> int:
-        return self.value.__hash__()
-
-
-class LabelWithConstraints(Label):
-    _KEY_CONSTRAINT = "KeyConstraint"
-
-    def __init__(self, value: str):
-        super(LabelWithConstraints, self).__init__(self._KEY_CONSTRAINT, value)
 
+@dataclass(unsafe_hash=True)
+class ClassificationLabel(Label):
+    key = field(default="Classification", init=False)
```

Comment:
Whoops, I forgot to include `:str` in the thing I pasted you

Reply:
fixed this.